src/applications/personalization/dashboard-2/components/health-care/HealthCare.jsx

```diff
@@ -150,19 +181,25 @@ const mapStateToProps = state => {
     prescriptions,
     canAccessRx,
     authenticatedWithSSOe: isAuthenticatedWithSSOe(state),
+    canAccessMessaging: profileState.services.includes(
+      backendServices.MESSAGING,
+    ),
+    unreadMessagesCount: selectUnreadMessagesCount(state),
   };
 };
 
 const mapDispatchToProps = {
+  fetchFolder: fetchFolderAction,
```

Comment:
idea: what if we made this `fetchInbox: () => {fetchFolderAction(0)}`? I think it would make the `useEffect` hook at the top of this component read a little bit nicer :)